complaints/templates/complaints/handle_complaints.html

```diff
@@ -39,8 +39,7 @@
                <tbody bgcolor="white">
                   <tr>
                      <td>{{ comp.category }}</td>
-
-                     <td>{{ comp.room_no }}</td>
+                     <td>{{ comp.room_no }}, {{ comp.bhavan }}</td>
```

Comment:
Bhavan should come before Room Number.

Reply:
Moreover, make sure that the complaint ids are also being displayed to users in the my complaint tab(I guess this is not being done) and to all the staff in All Complaints tab.